src/Compiler/pars.fsy

```diff
@@ -521,6 +521,12 @@ moduleIntro:
         let mModule = rhs parseState 1
         mModule, $4, $5.LongIdent, $3, $2 }
 
+  | moduleKeyword opt_attributes opt_access opt_rec OBLOCKSEP
+      { if not (isNil $2) then
+            parseState.LexBuffer.CheckLanguageFeatureAndRecover LanguageFeature.AttributesToRightOfModuleKeyword (rhs parseState 4)
```

Comment:
Do we have a test for this logical condition around LanguageFeature.AttributesToRightOfModuleKeyword somewhere?

Reply:
Yes, it looks like that some tests were added with the original rule in https://github.com/dotnet/fsharp/pull/11722.